c/expr/reduceop.cc

```diff
@@ -77,6 +78,24 @@ static void sum_skipna(const int32_t* groups, int32_t grp, void** params) {
 
 
 
+
+//------------------------------------------------------------------------------
+// Count calculation
+//------------------------------------------------------------------------------
+
+template<typename IT, typename OT>
+static void count_skipna(const int32_t* groups, int32_t grp, void** params) {
```

Comment:
Traditionally, `count(x)` function returns the number of non-NA values in `x`.
In your implementation, however, the function doesn't count anything, but merely returns the number of elements in each group. It's a valid function, just not a suitable name...

Reply:
Reimplemented this function to match the existing name.